ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
> So how should the version be specified for these jobs that are running pulp-smash?

I think that depends a lot on how ansible is being called. For example, we override a default value in our [Vagrantfile](https://github.com/pulp/pulp/blob/master/Vagrantfile.example#L40), because that's the best place for customizing ansible's behavior when vagrant calls it.

I've never worked with ansible in a CI setting so I don't know enough to say what's likely to be easiest.

> If no value is given, then Pulp Smash will assume that Pulp is the absolute latest development version. Do you think that's a sane default? We could do something with an interactive fact as you mention, though I don't know if either elyezer or I know how to do that.

It seems like the default should be a null (or nil) value, rather than not setting the var. And yes, I think that if it's not specified then defaulting to the latest is a great choice.